lib/add-exclude.ts

```diff
@@ -1,6 +1,6 @@
-module.exports = addExclude;
+export default addExclude;
```

Comment:
Not blocking: `export default` is a matter of personal preference and coding style but it can make it unclear which parts of a module are being exported, particularly when multiple defaults are exported from the same file.

Default exports, allow you to export a single functions from a module without using a specific name and then, when importing a default export, any name can be chosen. Sometimes it's a bit confusing when it happens.

Of course, if we chose to use named exports, we'll need to update the imports, for example from `import addExclude from './add-exclude';` to `import { addExclude } from './add-exclude';` -- out of scope of this PR I believe